CHANGES/6950.bugfix

```diff
@@ -0,0 +1,7 @@
+Increased the upper boundary of the :doc:`multidict:index` dependency
+to allow for the version 6 -- by :user:`hugovk`.
+
+It used to be limited below version 7 in :doc:`aiohttp <index>` v3.8.1 but
+was lowered in v3.8.2 via :pr:`6550` and never brought back, causing
+the upgrading users problems with their dependency pins. :doc:`aiohttp
+<index>` v3.8.3 fixes that by recovering the original boundary of ``< 7``.
```

Comment:
@hugovk WDYT about the change note update? The preview is here: https://aiohttp--6950.org.readthedocs.build/en/6950/changes.html#to-be-included-in-vrelease-if-present.

Reply:
Looks good, how about changing:

> causing the upgrading users problems with their dependency pins.

to:

> causing problems with dependency pins when upgrading.

?

Reply:
I agree with the below one too. Not only user-specified dependency pins but also other dependencies' (e.g. `bedevere`) dependency pins.

Reply:
Updated!

Reply:
Thanks!